src/main/java/com/iexec/common/replicate/ReplicateStatusUpdate.java

```diff
@@ -16,29 +16,29 @@
 
 package com.iexec.common.replicate;
 
-import java.util.Date;
-
 import com.fasterxml.jackson.annotation.JsonIgnore;
-
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import static com.iexec.common.replicate.ReplicateStatusModifier.*;
+import java.util.Date;
+
+import static com.iexec.common.replicate.ReplicateStatusModifier.POOL_MANAGER;
```

Comment:
```suggestion
import static com.iexec.common.replicate.ReplicateStatusModifier.*;
```
?

Reply:
For those imports, I tend to let IDE decide.
An implicit rule seems to be something like:
if there are less than 5 imports, import should be explicitly defined and wildcards should be avoided.

Reply:
I agree, keep the default, we'll re-talk about that if I found we don't have same defaults